internal/k8s/appprotectdos/app_protect_dos_configuration.go

```diff
@@ -269,7 +356,12 @@ func (fc *FakeConfiguration) DeletePolicy(key string) (changes []Change, problem
 	return changes, problems
 }
 
-// DeleteLogConf deletes an App Protect Dos Log Configuration from App Protect Dos Configuration
+// DeleteLogConf deletes an App Protect Dos Portected resource from App Protect Dos Configuration
```

Comment:
Typo and I'm not sure it's correct? (DeleteLogConf delete a protected resource?)

Reply:
fixed